dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/LinkageChecker.java

```diff
@@ -266,7 +266,11 @@ private boolean problemFilter(LinkageProblem linkageProblem) {
       if (!isClassAccessibleFrom(targetJavaClass, sourceClassName)) {
         AccessModifier modifier = AccessModifier.fromFlag(targetJavaClass.getModifiers());
         return Optional.of(
-            new InaccessibleClassProblem(sourceClassFile, targetClassFile, symbol, modifier));
+            new InaccessibleClassProblem(
+                sourceClassFile,
+                targetClassFile,
+                new ClassSymbol(symbol.getClassBinaryName()),
```

Comment:
The symbol for `InaccessibleClassProblem` should be a class symbol, not a field symbol or method symbol.